.doctor-rst.yaml

```diff
@@ -96,6 +100,5 @@ whitelist:
         - '.. versionadded:: 3.6' # MonologBundle
         - '// bin/console'
         - 'End to End Tests (E2E)'
-        - '.. code-block:: php'
```

Comment:
We fixed rule on https://github.com/OskarStark/doctor-rst/pull/1240, so we can remove it and show real remaining issues  :smiley: